content/blog/monthly-january-2020.md

```diff
@@ -0,0 +1,68 @@
+---
+title: This Month in TiKV - January 2020
+date: 2020-02-18
+author: Calvin Weng
+---
+
+<!-- Fill in the below from the API: https://developer.github.com/v4/explorer/ -->
+
+Welcome to the first monthly wrap-up for 2020. As folks in China are fighting against the outbreak of coronavirus, we keep our fingers crossed and hope this could end soon. In the meantime, we stay healthy and focus on driving forward our project.
+
+
+## Releases
+
+This month our team made 2 TiKV releases (3.0.9 and 4.0.0-beta). The 3.0.9 release includes the configuration change to speed up the Region scattering, some optimized configuration items and a fixed issue. The 4.0.0-beta release includes major updates such as quick backup and restoration support, RocksDB version upgrade，and also bugs fixes.
+
+You can review the changelogs here:
+
+*   [3.0.9](https://github.com/tikv/tikv/releases/tag/v3.0.9)
+*   [4.0.0-beta](https://github.com/tikv/tikv/releases/tag/v4.0.0-beta)
+
+## Reading materials
+
+[@brson](https://github.com/brson) published his first episode of the Rust Compile time series, [The Rust Compilation Calamity](https://tikv.org/blog/rust-compilation-calamity/), and shared his research and experiences with Rust compile times, using the TiKV project as a case study.
+
+[@BusyJay](https://github.com/BusyJay) (the first author) together with [@zhexuany](https://github.com/zhexuany) and [@bb7133](https://github.com/bb7133) (second authors) wrote [an overview on Raftstore in the series of A TiKV Source Code Walkthrough (in Chinese)](https://pingcap.com/blog-cn/tikv-source-code-reading-17/). This article uses the code of TiKV 3.0 version as an example to present the key definitions and designs in the raftstore source code.
+
+## Notable PRs
```

Comment:
If there is time, then it would be nice to explain all the terms here which readers might not understand: yatp, green GC, and the external_storage crate

Reply:
@yiwu-arbug Can you describe what is  "external_storage crate", and @MyonKeminta can you let me know what is Green Garbage Collection? Thanks

Reply:
Green GC's goal is to avoid GC wakes up the hibernating regions.
In the old GC procedure, during ResolveLocks phase, it needs to scan all regions to ensure all locks with ts <= safe_point need to be resolved. However the scanning wakes hibernating regions up.
In brief, Green GC changes the way to scan locks so that inactive regions won't be waken up from hibernating. By the way, there is also a PR([#6333](https://github.com/tikv/tikv/pull/6333)) that aoivds unnecessary waking up in the final GC phase.